integration/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     // Google dependencies
     constraints {
         implementation 'com.google.guava:guava:33.0.0-jre'
-        implementation 'org.json:json:20231013' // security vulnerability
+        implementation 'org.json:json:20240205' // security vulnerability
```

Comment:
We could consider removing the constraint and seeing what it uses without the explicit constraint.